### DISCUSSION TITLE
Connectors probe

### DISCUSSION THREAD

**Discussion by adeilme** [psi-probe/psi-probe#3503] [General]
Gentlemen in the tabs connectors of the probe I can no longer view the processes. should configure to exit the list of processes from the connector?
![Captura de tela 2024-06-30 110853](https://github.com/psi-probe/psi-probe/assets/169293379/d189b06f-706c-4f2d-af89-82b441b599b8)

**Comment by hazendaz** [comment]
Can you do me a favor, drop back to 4.x line and try that again to see if the issue is the same.  If so it affects both javax and jakarta. If not, that allows focus on just jakarta which should be easy to figure out.

**Comment by hazendaz** [comment]
this is working, closing as no follow up and check shows page working.
